### PR TITLE
feat(Chat): Revamp design.

### PR DESCRIPTION
**Pull request by muscat1**
* ensure keyboard stays open when sending messages on mobile web.

### CODE REVIEW

**Comment by muscat1** [comment]
Mobile:
Screenshot 2021-03-16 at 11 59 57

Desktop:
Screenshot 2021-03-16 at 12 00 21

File: react/features/chat/middleware.js [hunk 1]
```diff
@@ -81,16 +79,7 @@ MiddlewareRegistry.register(store => next => action => {
         break;
 
     case OPEN_CHAT:
-        if (localParticipant.name) {
-            dispatch(setActiveModalId(CHAT_VIEW_MODAL_ID));
-            _maybeFocusField();
-        } else {
-            dispatch(openDisplayNamePrompt(() => {
```
Comment (muscat1) [inline react/features/chat/middleware.js:88]: This seemed redundant to me, as we already display the `<DisplayNameForm />` inside the `<Chat />` component and it serves the same purpose. If it had a different reasoning, please let me know and I'll add it back.

Comment (saghul) [reply react/features/chat/middleware.js:88]: Looks good.

File: react/features/base/icons/svg/smile.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+<path fill-rule="evenodd" clip-rule="evenodd" d="M9.99996 18.3333C5.39759 18.3333 1.66663 14.6023 1.66663 9.99996C1.66663 5.39759 5.39759 1.66663 9.99996 1.66663C14.6023 1.66663 18.3333 5.39759 18.3333 9.99996C18.3333 14.6023 14.6023 18.3333 9.99996 18.3333ZM9.99996 16.6666C13.6819 16.6666 16.6666 13.6819 16.6666 9.99996C16.6666 6.31806 13.6819 3.33329 9.99996 3.33329C6.31806 3.33329 3.33329 6.31806 3.33329 9.99996C3.33329 13.6819 6.31806 16.6666 9.99996 16.6666ZM8.33329 8.74996C8.33329 9.44031 7.77365 9.99996 7.08329 9.99996C6.39294 9.99996 5.83329 9.44031 5.83329 8.74996C5.83329 8.0596 6.39294 7.49996 7.08329 7.49996C7.77365 7.49996 8.33329 8.0596 8.33329 8.74996ZM12.9166 9.99996C13.607 9.99996 14.1666 9.44031 14.1666 8.74996C14.1666 8.0596 13.607 7.49996 12.9166 7.49996C12.2263 7.49996 11.6666 8.0596 11.6666 8.74996C11.6666 9.44031 12.2263 9.99996 12.9166 9.99996ZM9.99996 15C11.7491 15 13.2885 14.1018 14.1821 12.7414L12.8867 11.6691C12.3021 12.6792 11.2131 13.3333 9.99996 13.3333C8.801 13.3333 7.71376 12.6771 7.12376 11.6672L5.8304 12.741C6.72576 14.0962 8.25894 15 9.99996 15Z" fill="white"/>
```
Comment (saghul) [inline react/features/base/icons/svg/smile.svg:2]: remove the fill attribute

**Review by saghul — COMMENTED** [review]
LGTM overall, but I'd like some more experienced CSS eyes on it. Also, by looking at the screenshots I think the emoji picker should be above the input field and not cover it, looks a bit odd.

File: react/features/chat/middleware.js (hunk 1 above)
Comment (saghul) [reply react/features/chat/middleware.js:88]: On a second look, nope. Mobile does not have that.

**Comment by vp8x8** [comment]
The CSS part looks good to me...

**Review by saghul — DISMISSED** [review]
LGTM but please remove the unrelated change.

**Review by saghul — APPROVED** [review]

**Pull request merged**